test/js_test.go

```diff
@@ -2343,15 +2360,31 @@ func TestJetStreamImportDirectOnly(t *testing.T) {
 		}
 	}
 
-	// Cannot subscribe with JS context from another account right now.
-	if _, err := js.SubscribeSync("ORDERS"); err != nats.ErrJetStreamNotEnabled {
-		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrJetStreamNotEnabled, err)
+	// Cannot create ephemeral subscriber with JS context.
+	if _, err := js.SubscribeSync("ORDERS", nats.BindOnly()); err != nats.ErrDurableNameRequired {
```

Comment:
I think we should be able to bind to any pre-existing consumer.

Reply:
It is possible to bind to ephemerals as well but have to use `nats.Durable` to set the consumer name

Reply:
See above.